genai-function-calling/vercel-ai/package.json

```diff
@@ -10,15 +10,9 @@
     "start": "node --env-file .env -r @elastic/opentelemetry-node index.js"
   },
   "dependencies": {
-    "ai": "^4.1.63",
-    "@ai-sdk/azure": "^1.2.6",
-    "@ai-sdk/openai": "^1.2.6",
-    "@elastic/opentelemetry-node": "*"
-  },
-  "_comment": "Override to avoid punycode warnings in recent versions of Node.JS",
```

Comment:
since this project doesn't require old node, we don't need to use node-fetch, so can avoid this cruft